sql/handler.h

```diff
@@ -994,6 +1012,46 @@ struct handlerton
                                     const char *table_name,
                                     bool is_sql_layer_system_table);
 
+  /**
+    Creates a new compression dictionary with the specified data for this SE.
+
+    @param hton                       handletron object.
+    @param thd                        thread descriptor.
+    @param name                       compression dictionary name
+    @param name_len                   compression dictionary name length
+    @param data                       compression dictionary data
+    @param data_len                   compression dictionary data length
+
+    @return HA_CREATE_ZIP_DICT_OK             - on success
```

Comment:
Maybe can refer to enum handler_create_zip_dict_result definition and description instead of repeating all values with their meanings here